wordpress/wp-content/plugins/sil-dictionary-webonary/include/searchform_func.php

```diff
@@ -250,7 +276,6 @@ function theCursorPosition(ofThisInput) {
 		}
 		echo $numberOfEntriesText;
 		echo "<br>";
-		$lastEditDate = $wpdb->get_var("SELECT post_date FROM " . $wpdb->posts . " WHERE post_status = 'publish' AND post_type = 'post' ORDER BY post_date DESC");
 
 		if(isset($lastEditDate) && $lastEditDate != "0000-00-00 00:00:00")
```

Comment:
I know this line didn't change, but if it's possible for the `$lastEditDate` to be null, then this line should be 
```
if(!empty($lastEditDate) && $lastEditDate != "0000-00-00 00:00:00")
```

Reply:
thanks! done.